Enhance extension download instructions with command option and example (#7775)
microsoft/vscode-docs: docs/editor/extension-marketplace.md

### Can I download an extension directly from the Marketplace?

Some users prefer to download an extension once from the Marketplace and then install it to multiple VS Code instances from a local share. This is useful when there are connectivity concerns or if your development team wants to use a fixed set of extensions.

> To download an extension, search for it in the Extensions view, right-click on an extension from the results, and select **Download VSIX**. Once downloaded, you can then install the extension via the **Install from VSIX** command in the Extensions view command dropdown.

Reviewer: ```suggestion
To download an extension, search for it in the Extensions view, right-click on an extension from the results, and select **Download VSIX**.
```

### Can I download an extension directly from the Marketplace?

Some users prefer to download an extension once from the Marketplace and then install it to multiple VS Code instances from a local share. This is useful when there are connectivity concerns or if your development team wants to use a fixed set of extensions.

> To download an extension, search for it in the Extensions view, right-click on an extension from the results, and select **Download VSIX**. 

Reviewer: ```suggestion
To download an extension, search for it in the Extensions view, right-click on an extension from the results, and select **Download VSIX**. Once downloaded, you can then install the extension via the **Install from VSIX** command in the Extensions view command dropdown.
```

> [!NOTE]
> The download option is only available for extensions that are not already installed. This experience is currently available in VS Code Insiders and will be available in VS Code Stable in the November release (1.96).

> Once downloaded, you can then install the extension via the **Install from VSIX** command in the Extensions view command dropdown.

Reviewer: ```suggestion
```

### Can I download an extension directly from the Marketplace?

Some users prefer to download an extension once from the Marketplace and then install it to multiple VS Code instances from a local share. This is useful when there are connectivity concerns or if your development team wants to use a fixed set of extensions.

> To download an extension, use the following command option:

Reviewer: ```suggestion
To download an extension, use the `--download-extension` command-line option and provide the extension ID and download location:
```

```
code --download-extension <UniqueExtensionID> --location <Path>

Example:
code --download-extension ms-python.python --location ~/vscode/testing
```

> You can copy the extension ID from the extension details page ![Copy Extension ID](./images/extension-marketplace/copy-extension-id.png).

Reviewer 2: Let's remove the "you can copy the extension ID" sentence. This does not have to be super easy.

Also let's add a note, something like "--download-extension is available in VS Code Insiders, and will be in VS Code Stable in December."

Reviewer: @isidorn I thought the plan was to release this guidance with the December release anyway?

Reviewer 2: I would release this guidance before, so that Marketplace can go ahead with their plan of removing the download URL before the Stable VS Code is out.

### Can I download an extension directly from the Marketplace?

Some users prefer to download an extension once from the Marketplace and then install it to multiple VS Code instances from a local share. This is useful when there are connectivity concerns or if your development team wants to use a fixed set of extensions.

> To download an extension, search for it in the Extensions view, right-click on an extension from the results, and select **Download VSIX**. Note that the download option is only available for extensions that are not already installed.

Reviewer: ```suggestion
To download an extension, search for it in the Extensions view, right-click on an extension from the results, and select **Download VSIX**. 

> [!NOTE]
> The download option is only available for extensions that are not already installed. This experience is currently available in VS Code Insiders and will be available in VS Code Stable November release (1.96).
```
